cms/static/coffee/src/keyword_validator.coffee

```diff
@@ -0,0 +1,28 @@
+define ->
```

Comment:
New code/files should not be written in Coffeescript, which has been deprecated in our codebase.